src/main/webapp/WEB-INF/views/admin/products.jsp

```diff
@@ -2,35 +2,45 @@
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
 <html>
 <head>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
     <title>All users</title>
 </head>
 <body>
-<table border="1">
-    <tr>
-        <th>ID</th>
-        <th>NAME</th>
-        <th>PRICE</th>
-        <th>DELETE</th>
-    </tr>
-    <c:forEach var="product" items="${products}">
+<table class="table">
+    <thead class="thead-dark">
         <tr>
-            <td>
-                <c:out value="${product.id}"/>
-            </td>
-            <td>
-                <c:out value="${product.name}"/>
-            </td>
-            <td>
-                <c:out value="${product.price}"/>
-            </td>
-            <td>
-                <a href="/internet_shop_war_exploded/admin/product/delete?productId=${product.id}">DELETE</a>
-            </td>
+            <th scope="col">ID</th>
+            <th scope="col">NAME</th>
+            <th scope="col">PRICE</th>
+            <th scope="col"></th>
         </tr>
+    </thead>
+    <c:forEach var="product" items="${products}">
+        <tbody>
+            <tr>
+                <td>
+                    <c:out value="${product.id}"/>
+                </td>
+                <td>
+                    <c:out value="${product.name}"/>
+                </td>
+                <td>
+                    <c:out value="${product.price}"/>
+                </td>
+                <td>
+                    <a href="/internet_shop_war_exploded/admin/product/delete?productId=${product.id}">DELETE</a>
```

Comment:
use context path instead of internet_shop_war_exploded. See example in the video

Reply:
done